src/ResourceManager/Compute/Commands.Compute/Generated/Snapshot/Config/NewAzureRmSnapshotUpdateConfigCommand.cs

```diff
@@ -121,76 +122,11 @@ protected override void ProcessRecord()
 
         private void Run()
         {
-            // CreationData
-            Microsoft.Azure.Management.Compute.Models.CreationData vCreationData = null;
-
             // EncryptionSettings
             Microsoft.Azure.Management.Compute.Models.EncryptionSettings vEncryptionSettings = null;
 
-#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
same question as above, how is the behavior altered if these obsoleted values are provided?